akka-cluster/src/multi-jvm/scala/akka/cluster/MultiNodeClusterSpec.scala

```diff
@@ -26,34 +26,43 @@ import akka.actor.RootActorPath
 
 object MultiNodeClusterSpec {
 
+  /** run all tests using artery, rather than the "old" remoting */
+  val useArtery = sys.props.get("akka.cluster.test.use-artery").contains("true")
+  def arteryEnablerFragment = ConfigFactory.parseString("akka.remote.artery.enabled=on")
```

Comment:
Hmm, yes, I think you are right that just enable it, I expected there to be more stuff that needs to change initially but I haven't found any places where there is yet.